### PR TITLE
task7

### PR DESCRIPTION
**Pull request by Liutianpin**
## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

**Comment by Liutianpin** [comment]
Task8 finished. The PR already covered the scope of task8.

File: typescript/simple-mercari-web/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
-USER trainee
+COPY package.json package.json
+COPY package-lock.json package-lock.json
+COPY tsconfig.json tsconfig.json
+COPY src src
+COPY public public
+RUN chmod -R 755 /app
+RUN npm ci
+EXPOSE 3000
+#RUN addgroup -S mercari && adduser -S trainee -G mercari
```
Comment (rhomel) [inline typescript/simple-mercari-web/Dockerfile:13]: nit: a better security practice is to run commands as a non-root user in a container.

File: typescript/simple-mercari-web/src/components/ItemList/ItemList.tsx [hunk 2]
```diff
@@ -46,12 +46,12 @@ export const ItemList: React.FC<Prop> = (props) => {
...
 
   return (
-    <div>
+    <div className='ItemsWrapper'>
       {items.map((item) => {
         return (
           <div key={item.id} className='ItemList'>
             {/* TODO: Task 1: Replace the placeholder image with the item image */}
-            <img src={placeholderImage} />
+            <img className='ItemImage' src={`${server}/image/${item.image_name}`} />
```
Comment (rhomel) [inline typescript/simple-mercari-web/src/components/ItemList/ItemList.tsx:54]: nit: in my opinion the previous name `image_filename` was more useful

**Review by rhomel — APPROVED** [review]
👍

**Pull request merged**